File: Project Euler/1-9/problem4.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import sys

max = 0
for i in reversed(xrange(101, 1000)):
    for j in reversed(xrange(101, 1000)):
        if i*j < max:
            break
        if str(i*j)==str(i*j)[::-1] and i*j > max:
            max = i*j
print max

File: Project Euler/1-9/ruby/problem4.rb
#!/usr/bin/env ruby
# encoding: utf-8

max = 0

def isPalindrome(string)
  return string == string.reverse
end

for i in 100..999
  for j in 100..999
    if isPalindrome((i*j).to_s) and i*j > max
      max = i*j
    end
  end
end

puts max
